display_list/display_list_builder.cc

```diff
@@ -794,7 +769,7 @@ bool DisplayListBuilder::quickReject(const SkRect& bounds) const {
   }
   SkRect dev_bounds;
   matrix.mapRect(bounds).roundOut(&dev_bounds);
-  return !current_layer_->clip_bounds().intersects(dev_bounds);
+  return !tracker_.device_cull_rect().intersects(dev_bounds);
```

Comment:
Perhaps add `Tracker::device_content_culled`? Or `Tracker::device_bounds_culled`? It doesn't save much, but it reads better in the calling code.

Reply:
It looks like we can directly call the existing `Tracker::content_culled` method.